### DISCUSSION TITLE
Necessary for Pixel

### DISCUSSION THREAD

**Discussion by aagha** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#363] [Q&A]
Is there bloatware on a Pixel device?

I'm curious if anyone has tried this on Pixel devices and if anything if found that's worth removing.

**Comment by AnonymousWP** [comment]
Bloatware, no, but they of course do come with applications in all sort of forms. See https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pulls?q=pixel for details.
